airbyte-integrations/connectors/source-mongodb-strict-encrypt/src/main/java/io.airbyte.integrations.source.mongodb/MongodbSourceStrictEncrypt.java

```diff
@@ -4,11 +4,17 @@
 
 package io.airbyte.integrations.source.mongodb;
 
+import static io.airbyte.integrations.source.mongodb.MongoDbSourceUtils.*;
```

Comment:
There's a setting for code style within IntelliJ where the usage of `*` is not preferred. You may need to update your IntelliJ [settings](https://airbyte.gitbook.io/airbyte/contributing-to-airbyte/code-style)